### PR TITLE
fix: test bindings on x86_64-unknown-linux-musl

### PR DESCRIPTION
**Pull request by yisibl**
`mimalloc-rust` is no longer maintained, migrated to `mimalloc`.

`local_dynamic_tls` must be enabled in Linux, otherwise the following error will occur:

```bash
Error: Error relocating /build/resvgjs.linux-x64-musl.node: mprotect: initial-exec TLS resolves to dynamic definition in /build/resvgjs.linux-x64-musl.node

  › Object.<anonymous> (js-binding.js:175:31)
  › Module._compile (node_modules/pirates/lib/index.js:117:24)

  ✘ __test__/index.spec.ts exited with a non-zero exit code: 1
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -23,8 +23,11 @@ pathfinder_simd = { version = "0.5.2", features = ["pf-no-simd"] }
 futures = "0.3.21"
 woff2 = "0.3.0"
 
-[target.'cfg(all(not(all(target_os = "linux", target_arch = "aarch64", target_env = "musl")), not(all(target_os = "windows", target_arch = "aarch64")), not(target_arch = "wasm32")))'.dependencies]
-mimalloc-rust = { version = "0.2" }
+[target.'cfg(not(any(target_os = "linux", target_family = "wasm")))'.dependencies]
+mimalloc = "0.1"
+
+[target.'cfg(target_os = "linux")'.dependencies]
+mimalloc = { version = "0.1", features = ["local_dynamic_tls"] }
```
Comment (yisibl) [inline Cargo.toml:30]: 迁移到 mimalloc 测试就好了？

Comment (Brooooooklyn) [reply Cargo.toml:30]: 你需要在 Linux 下开启 `local_dynamic_tls`

Comment (Brooooooklyn) [reply Cargo.toml:30]: 支持, 但是不维护了

Comment (yisibl) [reply Cargo.toml:30]: 感谢！

**Pull request merged**